### ISSUE TITLE
[Bug] visitor log icon tooltips not readable by NVDA screen reader on desktop Windows

### ISSUE THREAD

**Issue by diosmosis** [piwik/piwik#23175]
### What happened?

It seems the visitor log icon tooltips are not detected by NVDA (see https://wordpress.org/support/topic/problem-with-open-visitor-profile-with-screen-reader/#post-18377007). I suspect this is due to the images not having an alt text and due to the tooltip adding an aria-describedby attribute temporarily on hover (which may not be possible to do with every screen reader).

### What should happen?

The icons and their tooltips should be accessible to users that use a screen reader.

### How can this be reproduced?

On Windows with NVDA installed, try to navigate to the visitor log icons and have their tooltips read out loud.

### Matomo version

5.3.0

### Computer operating system

Windows

### Relevant log output

```shell

```

### Validations

- [x] Read our [Contributing Guidelines](https://github.com/matomo-org/matomo/blob/5.x-dev/CONTRIBUTING.md).
- [x] Follow our [Security Policy](https://github.com/matomo-org/matomo/blob/5.x-dev/SECURITY.md).
- [x] Check that there isn't already an issue that reports the same bug to avoid creating duplicates.
- [x] The provided steps to reproduce is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the Bug.
